src/test/java/seedu/address/logic/commands/TagCommandTest.java

```diff
@@ -133,16 +131,16 @@ public void equals() {
         assertTrue(command1.equals(command1));
 
         // different objects, same fields -> returns true
-        assertEquals(command1, command2);
+        assertTrue(command1.equals(command2));
```

Comment:
Not sure why author changed from `assertEquals` to `assertTrue`. As `assertEquals` would provide better readability.